src/Equinox/Flow.fs

```diff
@@ -11,21 +11,22 @@ type StreamToken = { value : obj; version: int64 }
 /// Internal type used to represent the outcome of a TrySync operation
 [<NoEquality; NoComparison; RequireQualifiedAccess>]
 type SyncResult<'state> =
-    /// The write succeeded (the supplied token and state can be used to efficiently continue the processing iff desired)
+    /// The write succeeded (the supplied token and state can be used to efficiently continue the processing if desired)
```

Comment:
these `iff`s are intentional - meaning if and only if - attempting to convey - this is not just a nice trick, its important only if desired

Reply:
oh, my bad, I didn't know it was an abbreviation